### PR TITLE
trigger CI for track2 when tools is changed

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Trigger CI for core and track2 packages when tools is changed.

### CODE REVIEW

File: sdk/appconfiguration/ci.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ trigger:
     include:
     - sdk/appconfiguration/
     - sdk/core/
+    - tools/
```
Comment (scbedd) [inline sdk/appconfiguration/ci.yml]: Was `eng/` deliberately excluded here before? `tools/` is what is being asked for, but we should also be doing it for `eng/` correct?

Comment (praveenkuttappan) [reply sdk/appconfiguration/ci.yml]: I had missed it earlier. Thanks for the comment. I have fixed all yml file now.

**Comment by weshaggard** [comment]
@praveenkuttappan @scbedd is there a reason we want to trigger ci for all these pipelines if anything under eng changes? We generally use one pipeline like core as a proxy for those changes to prevent all pipelines from triggering. I noticed while merging https://github.com/Azure/azure-sdk-for-python/pull/10761 that we ran a lot more ci pipelines then I expected to see which is what triggered me to find this PR.

File: sdk/appconfiguration/ci.yml (hunk 1 above)
Comment (weshaggard) [reply sdk/appconfiguration/ci.yml:25]: I'd suggest not triggering all based on eng changes.

**Comment by praveenkuttappan** [comment]
I believe it as per the discussion with Laurent we made change to trigger all track2 pipelines when anything inside eng is changed. I am trying to find that discussion thread. But we should probably add more granular control so it wont trigger all when anything in eng is changed. for e.g. eng/common.

**Pull request merged**